recipes/blis/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "spacy-blis" %}
```

Comment:
This should probably be called `cython-blis`, given that is what the authors called it.

Reply:
Also, as a matter of hygiene, the directory name should match the package name,

Reply:
Done for both.